ts/quill/util.ts

```diff
@@ -93,17 +93,14 @@ export const getTextAndMentionsFromOps = (
 };
 
 export const getBlotTextPartitions = (
-  blot: LeafBlot,
+  blotText: string | undefined,
   index: number
 ): [string, string] => {
-  if (blot !== undefined && blot.text !== undefined) {
-    const leftLeafText = blot.text.substr(0, index);
-    const rightLeafText = blot.text.substr(index);
-
-    return [leftLeafText, rightLeafText];
-  }
+  const lowerCaseBlotText = blotText?.toLowerCase();
+  const leftLeafText = lowerCaseBlotText?.substr(0, index);
+  const rightLeafText = lowerCaseBlotText?.substr(index);
 
-  return ['', ''];
+  return [leftLeafText ?? '', rightLeafText ?? ''];
```

Comment:
Unfortunately, our Storybook build doesn't support `??` yet. This is a bigger problem that we need to fix, but until then, could you change these `??`s to `||`? It should logically be the same.